packages/neuron-ui/src/utils/storage.ts

```diff
@@ -0,0 +1,4 @@
+export const getGuidedFlag = () => window.localStorage.getItem('guided')
```

Comment:
The create/import guide is not shown by checking if it's already displayed before. Instead, it will show when user has 0 wallets; or when user explicitly start creating/importing another wallet.